riff-raff/app/conf/context.scala

```diff
@@ -79,7 +78,6 @@ class Configuration(val application: String, val webappConfDirectory: String = "
     lazy val clientSecret: String = configuration.getStringProperty("auth.clientSecret").getOrException("No client secret configured")
     lazy val redirectUrl: String = configuration.getStringProperty("auth.redirectUrl").getOrElse(s"${urls.publicPrefix}${routes.Login.oauth2Callback().url}")
     lazy val domain: String = configuration.getStringProperty("auth.domain").getOrException("No auth domain configured")
-    lazy val googleAuthConfig = GoogleAuthConfig(auth.clientId, auth.clientSecret, auth.redirectUrl, auth.domain)
```

Comment:
Because the new version `GoogleAuthConfig ` needs a anti forgery checker, and the default one requires a `HttpConfiguration` from the context, the authentication config need to be created when components are created.